### PR TITLE
Update fun.py

### PR DESCRIPTION
**Pull request by skruglov2023**
Added a few ftc-specific attacks, as there's frc-specific ones included there. Also increased the start hp and added a few more attack damages Reduced dev/bot damage to 1.5x

### CODE REVIEW

File: dozer/cogs/fun.py [hunk 1]
```diff
@@ -74,30 +81,28 @@ async def battle(self, ctx: DozerContext, opponent: discord.Member, delete_resul
...
+        hps = [1500, 1500]
         turn = random.randint(0, 1)
 
         messages = []
         while hps[0] > 0 and hps[1] > 0:
             opp_idx = (turn + 1) % 2
             damage = random.choice(damages)
-            if players[turn].id in ctx.bot.config['developers'] or players[turn] == ctx.bot.user:
-                damage = damage * 2
+            if players[turn].id in ctx.bot.config['developers'] or players[turn] == ctx.bot.user or players[turn].id == 675726066018680861:
```
Comment (tweirtx) [inline dozer/cogs/fun.py:93]: Hardcoded ID, bad

File: dozer/cogs/fun.py [hunk 2]
```diff
@@ -111,7 +116,7 @@ async def battle(self, ctx: DozerContext, opponent: discord.Member, delete_resul
     @guild_only()
     @discord.ext.commands.cooldown(1, 5, BucketType.channel)
     @discord.ext.commands.max_concurrency(1, per=BucketType.channel, wait=False)
-    @command()
+    @commands.command()
```
Comment (tweirtx) [inline dozer/cogs/fun.py:119]: No

File: dozer/cogs/fun.py [hunk 3]
```diff
@@ -50,10 +56,11 @@ async def battle(self, ctx: DozerContext, opponent: discord.Member, delete_resul
             "**{opponent}** was knocked off the traversal bar by **{attacker}**",
             "**{opponent}** had their battery fall out out thanks to **{attacker}**",
             "**{opponent}** had their season ended by **{attacker}**",
+            "**{opponent}** had their control hub bricked by **{attacker}**",
             "**{opponent}** had their roborio bricked by **{attacker}**",
             # this and the following messages are thanks to J-Man from the CHS discord server, who expended their
             # creative powers on these statements.
-            "**{opponent}** extended too far outside their frame perimeter in front of **{attacker}**",
+            "**{opponent}** extended too far outside their field perimeter in front of **{attacker}**",
```
Comment (tweirtx) [inline dozer/cogs/fun.py:63]: Leave both of these in

**Review by tweirtx — CHANGES_REQUESTED** [review]

File: dozer/cogs/fun.py (hunk 2 above)
Comment (skruglov2023) [reply dozer/cogs/fun.py:120]: Pretty sure the @commands.command part comes from how discord.ext commands are being imported

**Review by tweirtx — APPROVED** [review]

**Pull request merged**